ethereum/core/src/jmh/java/org/hyperledger/besu/ethereum/vm/operations/OperationBenchmarkHelper.java

```diff
@@ -58,12 +61,18 @@ private OperationBenchmarkHelper(
 
   public static OperationBenchmarkHelper create() throws IOException {
     final Path storageDirectory = Files.createTempDirectory("benchmark");
-    final KeyValueStorage keyValueStorage =
-        new RocksDBKeyValueStorage(
+    final OptimisticRocksDBColumnarKeyValueStorage optimisticRocksDBColumnarKeyValueStorage =
+        new OptimisticRocksDBColumnarKeyValueStorage(
```

Comment:
Struggling to see the relevance of this change...KeyValueStorage still exists doesn't it? Or maybe we should delete KeyValueStorage too?

Reply:
looks like the change is it's now a two step process - KeyValueStorage (which is the interface used by plugins) is created from this storage

Reply:
yes we want to start a refactor of this part to be only segmented as we are only using that now. this one is the first PR and we will have more PRs after this one. We found a lot of code smell in this part when we wanted to change the code in order to have only one transaction to commit the worldstate